concepts/Common/Security Considerations/20 HTML Encoding/30 Potentially Dangerous API Members/0 Potentially Dangerous API Members.md

```diff
@@ -0,0 +1 @@
+DevExtreme components encode HTML in most cases. However, several components include API members that allow you to disable or bypass the encoding. The following sections describe these potentially vulnerable API members.
```

Comment:
I'm not sure about the word "dangerous". It looks very serious :) How about "vulnerable"? Also, we could drop the word "members" to make the title shorter.